File: Java/src/LengthOfLastWordTest.java
import org.junit.Assert;
import org.junit.Test;

public class LengthOfLastWordTest {

    @Test
    public void testLengthOfLastWord() {
        LengthOfLastWord llw = new LengthOfLastWord();

        Assert.assertTrue(llw.lengthOfLastWord("Hello World") == 5);

        Assert.assertTrue(llw.lengthOfLastWord(" ") == 0);

        Assert.assertTrue(llw.lengthOfLastWord("a") == 1);

        Assert.assertTrue(llw.lengthOfLastWord("a ") == 1);
    }
    
}

File: Kotlin/src/LengthOfLastWordTest.kt
import org.junit.Test

class LengthOfLastWordTest {

    @Test
    fun testLengthOfLastWord() {
        val llw = LengthOfLastWord()

        assert(llw.lengthOfLastWord("Hello World") == 5)

        assert(llw.lengthOfLastWord(" ") == 0)

        assert(llw.lengthOfLastWord("a") == 1)

        assert(llw.lengthOfLastWord("a ") == 1)
    }

}